### ISSUE TITLE
Camera Movment

### ISSUE THREAD

**Issue by Orkdavid** [keiwando/evolution#14]
Im currently playing on the IOS app version V3.1.0 and this is kind of feature request/bug report in one. Do I tried to create an nord. that flys, successfully.. but the camera dependent move up or down and so the bird always escaped into the sky. 
i would love to see the camera move up and down as well as left and right, in the future.

**Comment by keiwando** [comment]
Yes, this is something I want to look into once I have some time to work on this project again. Do you happen to have a saved simulation in which your creature reliably flies out of the top of the screen and if so, would you mind sharing that file with me? That would be very helpful!

**Comment by Orkdavid** [comment]
Final got to it. Sry for the late response
[First Bird.zip](https://github.com/keiwando/evolution/files/11422875/First.Bird.zip)

**Comment by keiwando** [comment]
Thank you!

**Comment by keiwando** [comment]
With the 4.0 update, the camera now follows creatures vertically as well.
